src/components/admin/ride-request/RideRequestList.tsx

```diff
@@ -0,0 +1,145 @@
+import React, { useState } from "react";
+import RideRequestItem from "./RideRequestItem";
+import DropDown from "@/components/common/admin/DropDown";
+import SearchBar from "@/components/common/admin/SearchBar";
+import Pagination from "@/components/common/admin/Pagination";
+
+const options = [
+  { value: "onroute", label: "onRoute" },
+  { value: "waiting", label: "Waiting" },
+  { value: "completed", label: "Completed" },
+  { value: "canceled", label: "Canceled" },
+  { value: "", label: "None" },
+];
+const opts = ["all", "onroute", "waiting", "completed", "canceled"];
+const RideRequestList = () => {
+  const [searchQuery, setSearchQuery] = useState("");
+
+  const [selectedValue, setSelectedValue] = useState("");
+  const [currentPage, setCurrentPage] = useState(1);
+
+  const handleSearch = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    setSearchQuery(event.currentTarget.search.value);
+  };
+
+  const handleInputChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setSearchQuery(event.target.value);
+  };
+
+  const handleSelect = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    const value = event.target.value;
+    setSelectedValue(value);
+  };
+
+  return (
+    <div>
+      <div className="flex w-full justify-evenly my-10 items-center">
+        <DropDown
```

Comment:
the search Bar Should come first

Reply:
the ui is the drop down first should I change it?